seed.js

```diff
@@ -108,7 +120,7 @@ var seedExperiences = function (categories, randomizerIdx) {
       experience.quantity = randomizerIdx(1, 10);
       experience.price = faker.commerce.price();
       experience.categories = [categories[randomizerIdx(0, 8)]];
-      experience.photoUrl = faker.image.nightlife();
+      experience.photoUrl = _photoUrls[randomizerIdx(0, 37)];
```

Comment:
@johnhillphoto Perhaps use `_photoUrls[randomizerIdx(0, _photoUrls.length - 1)]` as I'm sure you'll find more photos worth adding :)